At the moment only the "amd64" architecture is defined here but it should be
possible to add more architectures here.

--- platforms/BUILD.bazel.orig	2021-01-08 07:47:26 UTC
+++ platforms/BUILD.bazel
@@ -23,3 +23,11 @@ config_setting(
         "@platforms//cpu:x86_64",
     ],
 )
+
+config_setting(
+    name = "freebsd_x86_64",
+    constraint_values = [
+        "@platforms//os:freebsd",
+        "@platforms//cpu:x86_64",
+    ],
+)
